extensions/flight-sql/src/main/java/io/deephaven/server/flightsql/FlightSqlResolver.java

```diff
@@ -664,9 +660,10 @@ private Table executeSqlQuery(SessionState session, String sql) {
             if (!(node instanceof TicketTable)) {
                 continue;
             }
-            final String variableName = new String(((TicketTable) node).ticket(), StandardCharsets.UTF_8).substring(2);
-            final Table sourceTable = queryScopeTables.get(variableName);
-            Assert.neqNull(sourceTable, "sourceTable");
+            // We know that all TicketTables currently produced by Flight SQL will be global scope tickets
+            final Table sourceTable = scopeTicketResolver
```

Comment:
I think we should be going to `queryScopeTables`, not re-resolving from scratch, in order to avoid potentially repeating transform operations.